var-core/src/main/java/io/varhttp/VarServlet.java

```diff
@@ -95,10 +105,24 @@ public void handle(HttpServletRequest request, HttpServletResponse response) {
 		}
 	}
 
+
+
 	public void configure(Consumer<VarConfiguration> configuration) {
-		VarConfiguration varConfiguration = new VarConfiguration(this, controllerMapper, baseConfigurationContext, parameterHandler);
+		VarConfiguration varConfiguration = new VarConfiguration(this, controllerMapper, baseConfigurationContext, parameterHandler, registeredWebSockets, null);
 		configuration.accept(varConfiguration);
 		baseConfigurationContext.applyMappings();
 		varConfiguration.applyMappings();
 	}
+
+	@Override
+	public void destroy() {
+		super.destroy();
+		for (VarWebSocket webSocket : webSockets) {
+			try {
+				webSocket.close();
+			} catch (Exception exception) {
+				throw new RuntimeException(exception);
```

Comment:
Is it correct? Whenever there is an issue while closing a websocket, we prevent any other closures?

Reply:
removed